## debian-file-locations.diff by Jens Peter Secher <jps@debian.org>
##
## Set class path to ./haxe/std/ and /usr/share/haxe/.

Index: haxe/haxe/main.ml
===================================================================
--- haxe.orig/haxe/main.ml	2011-02-01 00:29:32.000000000 +0100
+++ haxe/haxe/main.ml	2011-02-01 01:03:48.000000000 +0100
@@ -277,7 +277,7 @@
 	with
 		Not_found ->
 			if Sys.os_type = "Unix" then
-				com.class_path <- ["/usr/lib/haxe/std/";"/usr/local/lib/haxe/std/";"";"/"]
+				com.class_path <- ["./haxe/std/";"/usr/share/haxe/std/";"";"/"]
 			else
 				let base_path = normalize_path (try executable_path() with _ -> "./") in
 				com.class_path <- [base_path ^ "std/";""]);
